packages/sw-precaching/demo/sw.js

```diff
@@ -0,0 +1,44 @@
+/* global goog */
+
+importScripts('./sw-precaching.v0.0.13.min.js');
```

Comment:
I've been pulling in libraries as `../build/my-lib.js` for the other demos. For production `importScript()` uses we want to encourage pulling in versioned libraries (or using a module bundler), but I wonder if it can lead to more trouble than it's worth to try to make sure that the right version of the library ends up in the `demo/` directory, and that we update it each time we push out a new version.

Also... I don't see `sw-precaching.v0.0.13.min.js` included in this PR. So maybe this isn't meant to work to begin with?